Framework/Core/src/WorkflowSerializationHelpers.cxx

```diff
@@ -277,34 +298,28 @@ struct WorkflowImporter : public rapidjson::BaseReaderHandler<rapidjson::UTF8<>,
   {
     enter("END_OBJECT");
     if (in(State::IN_INPUT)) {
-      if (!inputHasDescription && !inputHasSubSpec) {
-
-        DataDescriptorMatcher expectedMatcher00{
-          DataDescriptorMatcher::Op::And,
-          OriginValueMatcher{origin.str},
-          std::make_unique<DataDescriptorMatcher>(
-            DataDescriptorMatcher::Op::And,
-            DescriptionValueMatcher{ContextRef{1}},
-            std::make_unique<DataDescriptorMatcher>(
-              DataDescriptorMatcher::Op::And,
-              SubSpecificationTypeValueMatcher{ContextRef{2}},
-              std::make_unique<DataDescriptorMatcher>(DataDescriptorMatcher::Op::Just,
-                                                      StartTimeValueMatcher{ContextRef{0}})))};
-
-        dataProcessors.back().inputs.push_back(InputSpec({binding}, std::move(expectedMatcher00)));
```

Comment:
@ktf in this case, the `lifetime` and `inputOptions` parameters were missing, was this a bug? The two other cases below are using those two additional parameters.